packages/react/src/SignalWire/CoreVideo.tsx

```diff
@@ -56,57 +55,49 @@ export function CoreVideo({ ...props }: IVideoProps) {
         }
       }
 
-      const currentRoomSession = await props.client.createCall({
-        uri: props.address.channels.video,
+      console.log("Dialing to the address");
+      const currentCall = await props.client.dial({
+        to: props.address.channels.video,
 
         // @ts-expect-error undefined is not assignable to rootElement
-        rootElement: props.rootElement?.current ?? undefined,
+        // rootElement: props.rootElement?.current ?? undefined,
```

Comment:
@niravcodes just to understand, why do we need to comment this out?

Reply:
The new SignalWire interface expects rootElement to be registered at the time of instantiation and not during dial so we dont actually need the line there.